skyVegasTests/utilities/startEventListenerHelper.ts

```diff
@@ -0,0 +1,9 @@
+import { type Page } from "@playwright/test";
+
+export const startEventListener = (page:Page, consoleMessages:string[]) => {
```

Comment:
Do we want to add the new helper functions to index.ts which will act as a single entry point? If we dont want to club everything - might be worth removing that file so that each helper is isolated

Reply:
All done :)